examples/controls/package.json

```diff
@@ -1,19 +1,28 @@
 {
   "scripts": {
-    "start": "webpack-dev-server --progress --hot --open",
-    "start-local": "webpack-dev-server --env.local --progress --hot --open"
+    "start": "webpack-dev-server --progress --hot --port 3000 --open",
```

Comment:
Interestingly I recently approved a PR (perhaps in one of our other repos?) that removed this port specification, with the argument that it would let webpack-dev-server dynamically pick ports when running several examples. I haven't tested this extensively though, not sure which is better.